aws/resource_aws_eip.go

```diff
@@ -80,6 +80,13 @@ func resourceAwsEip() *schema.Resource {
 				Optional: true,
 			},
 
+			"public_ipv4_pool": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
```

Comment:
Instead of marking this as `Computed: true`, this seems like it should be `Default: "amazon"` to match the API.

Reply:
@mhorbul was able to confirm, thanks!

```
$ aws ec2 allocate-address --domain standard --public-ipv4-pool amazon

An error occurred (InvalidParameterCombination) when calling the AllocateAddress operation: Public IPv4 Pools are not available for EC2-Classic.
```